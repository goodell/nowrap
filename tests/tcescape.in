ordinary text
this[01mbold[0mtext
singleescape
twoescapeoccurrences
adjacentescapes
adjacentescapes
after singlespace
after   manyspaces
after	tabulator
endescape
endescap
