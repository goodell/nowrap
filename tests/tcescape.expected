ordinary t
this[01mbold[0mte
singleescap
twoescapeocc
adjacentesc
adjacentesc
after single
after   manysp
after	tab
endescape
endescap
